### PR TITLE
Add pycodestyle scripts to obs-monitor

### LINKED ISSUE #10: Add pycodestyle
Add `pycodestyle` to obs-monitor.

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
Added `pycodestyle_run.py` and related files (lifted from eva).  Renamed `plotObsMon `to `plotObsMon.py` so pycodestyle_run.py could pick it up.

Resolves #10

### CODE REVIEW

File: pycodestyle.cfg
Comment (CoryMartin-NOAA) [inline pycodestyle.cfg]: I know we "borrowed" from eva, but we can just remove the copyright stuff from these 2 files

**Review by CoryMartin-NOAA — COMMENTED** [review]
Interestingly enough, the GitHub actions norms didn't run on this PR... I think that was my error, should we fix that in this PR too? (I think the .github/workflows/norms.yaml file needs [push, pull_request] instead of just [push])

**Review by CoryMartin-NOAA — APPROVED** [review]
Thanks @EdwardSafford-NOAA !!

**Comment by EdwardSafford-NOAA** [comment]
Interestingly when running `pycodestyle_run.py` at the command line it didn't pick up `ush/plotObsMon`, which is why I renamed it to `plotObsMon.py`.  There is probably a way to instruct `pycodestyle_run.py` to run on files without the .py extension but this is working for both the CI and command line so I think we go with it as is.

**Comment by CoryMartin-NOAA** [comment]
@EdwardSafford-NOAA I snuck in a quick change to the `norms.yaml` to not explicitly run that file now that the name changed (otherwise it would fail). I think forcing `.py` is fine

**Comment by EdwardSafford-NOAA** [comment]
Yes, I spotted the explicit mention of `plotObsMon` in the `norms.yaml` file.  I made a similar change and committed it.  I'm a bit curious why github didn't offer any warning to me but no matter.  As long as things are the way we want them to be it's all good.

**Comment by CoryMartin-NOAA** [comment]
@EdwardSafford-NOAA great! I think we should wait for one more approval, then we can merge.

**Pull request merged**
